ext/standard/php_mail.h

```diff
@@ -28,12 +28,12 @@ do { \
 		php_mail_build_headers_elem(&s, key, val); \
 	} else if (Z_TYPE_P(val) == IS_ARRAY) { \
 		if (!strncasecmp(target, ZSTR_VAL(key), ZSTR_LEN(key))) { \
-			php_error_docref(NULL, E_WARNING, "'%s' header must be at most one header. Array is passed for '%s'", target, target); \
+			zend_type_error("Header \"%s\" must be of type string, array given", target); \
 			continue; \
```

Comment:
continue -> break?

Reply:
I'll change this for clarity, although It seems that `break` and `continue` behaves the same in this case, since we are in an inner `do { } while (0)` because of the macro.

Reply:
Or I can even remove it, if my hypothesis is right :)